tests/index.test.js

```diff
@@ -64,6 +66,34 @@ class ComponentStub {
   }
 }
 
+const Actions = {
+  submit: expect.createSpy(),
+}
+
+class FormComponent extends Component{
+  constructor(props){
+    super(props);
+
+    this.state = {};
+  }
+
+  render(){
+    return(
+      <form onSubmit={handleSumbit(Actions.submit)}>
+        <input name='testInput' type='text' defaultValue='value' />
+        <div>
+          <input name='nestedInput' type='text' defaultValue='nestedValue' />
+        </div>
+        <select name='testSelect'>
+          <option value='test1'>Test 1</option>
```

Comment:
Can you add two more options and have the middle selected? Right now it's just picking the first thing